### PR TITLE
Fix petition issues

### LINKED ISSUE #648: Fix 500 error when Petition Action goal = 0
Small bug:
Pages return a 500 error when the petition has goal=0 
https://act.staging.eff.org/action/keep-safe-harbors-safe-ed9c6cc0-8f55-4945-a760-659c7e145536

### PR DESCRIPTION
**Pull request by esoterik**
closes #648

### CODE REVIEW

**Comment by esoterik** [comment]
also closes #647 !

**Comment by lschatzkin** [comment]
LGTM!

File: app/views/tools/_petition.html.erb [hunk 1]
```diff
@@ -38,8 +37,7 @@
           <%= render 'tools/thank_you' -%>
         </div>
         <div class='unsigned'>
-          <%= form_for @signature, url: '/tools/petition',
-            html: {class: 'petition-tool'} do |f| -%>
+          <%= form_for @signature, url: tools_petition_path, html: {class: 'petition-tool'} do |f| -%>
```
Comment (lschatzkin) [inline app/views/tools/_petition.html.erb]: Just so I know, where is the tools_petition_path get set?

Comment (esoterik) [reply app/views/tools/_petition.html.erb]: all of these path helpers are created in [this file](https://github.com/EFForg/action-center-platform/blob/master/config/routes.rb#L12) (the line I linked to makes that one!)

`config/routes.rb` does more than that though -- it controls what URLs go to what controller methods. I often reference [this guide](https://guides.rubyonrails.org/routing.html) when I'm making routing changes.

Comment (lschatzkin) [reply app/views/tools/_petition.html.erb]: Cool, thank you!

**Pull request merged**